lib/meilisearch/client.rb

```diff
@@ -73,6 +73,16 @@ def stats
       http_get '/stats'
     end
 
+    ### DUMPS
+
+    def create_dump
+      http_post '/dumps'
+    end
+
+    def get_dump_status(dump_uid)
+      http_get "/dumps/#{dump_uid}/status"
+    end
```

Comment:
We would rather not use the `get_` as a prefix for a method. 
See the rule of rubocop: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/AccessorMethodName
I'm a little bit worried that rubocop does not notice it.
However, you can add an alias:

```ruby
    def dump_status(dump_uid)
      http_get "/dumps/#{dump_uid}/status"
    end
    alias get_dump_status dump_status
```

And if you add an alias, you should test it, see: https://github.com/meilisearch/meilisearch-ruby/blob/76986905012aa98749d162799a36aed60cdfdb43/spec/meilisearch/index/documents_spec.rb#L216-L225

Reply:
Thanks for the feedback. I've just corrected it.